variables.tf

```diff
@@ -148,3 +148,17 @@ variable "sse_algorithm" {
   description = "The server-side encryption algorithm to use"
   default     = "aws:kms"
 }
+
+variable "sns_arn" {
+  type        = string
+  description = "The arn for the SNS"
+  default     = ""
```

Comment:
`topic_arn` is a required attribute of the `aws_s3_bucket_notification` resource and setting the default to an empty string would error out, if the `sns_arn` was not supplied.